src/bootstrap/src/core/config/config.rs

```diff
@@ -1249,11 +1247,13 @@ impl Config {
         // has already been (kinda-cross-)compiled to Windows land, we require a normal Windows path.
         cmd.arg("rev-parse").arg("--show-cdup");
         // Discard stderr because we expect this to fail when building from a tarball.
-        let output = cmd
-            .stderr(std::process::Stdio::null())
-            .output()
-            .ok()
-            .and_then(|output| if output.status.success() { Some(output) } else { None });
+        let output = cmd.stderr(std::process::Stdio::null()).output().ok().and_then(|output| {
+            if output.status.success() {
+                Some(output)
+            } else {
+                None
+            }
+        });
```

Comment:
rust-analyzer autoformatting did this change - I guess we can keep it as tidy doesn't complain about it